### PR TITLE
Update action version

### PR DESCRIPTION
**Pull request by ipdae**
- Update latest lib9c & api

### CODE REVIEW

File: NineChronicles.Headless/GraphTypes/States/Models/Item/CostumeType.cs [hunk 1]
```diff
@@ -15,6 +15,10 @@ public CostumeType()
                 nameof(Costume.Equipped),
                 description: "Status of Avatar equipped."
             );
+            Field<NonNullGraphType<LongGraphType>>(
+                nameof(Costume.RequiredBlockIndex),
+                description: "Block index at the costume can use."
+            );
```
Comment (area363) [inline NineChronicles.Headless/GraphTypes/States/Models/Item/CostumeType.cs:21]: Can you elaborate on them meaning of `RequiredBlockIndex`? I think the description needs more detail but I'm not entirely sure what it means 😅 .

File: NineChronicles.Headless/GraphTypes/States/Models/Item/ShopItemType.cs [hunk 2]
```diff
@@ -35,6 +35,11 @@ public ShopItemType()
                 description: "Costume information.",
                 resolve: context => context.Source.Costume
             );
+            Field<NonNullGraphType<LongGraphType>>(
+                nameof(ShopItem.ExpiredBlockIndex),
+                description: "Block index at the shop item expiration.",
+                resolve: context => context.Source.ExpiredBlockIndex
+            );
```
Comment (area363) [inline NineChronicles.Headless/GraphTypes/States/Models/Item/ShopItemType.cs:42]: I'd appreciate it if you could explain `ExpiredBlockIndex` as well 🙇 .

File: NineChronicles.Headless/GraphTypes/States/Models/Item/CostumeType.cs (hunk 1 above)
Comment (ipdae) [reply NineChronicles.Headless/GraphTypes/States/Models/Item/CostumeType.cs:21]: This is locking item and can't be used if the current blockchain index is lower than the `RequiredBlockIndex `given to this item.

Comment (area363) [inline NineChronicles.Headless/GraphTypes/States/Models/Item/CostumeType.cs:21]:
```suggestion
            Field<NonNullGraphType<LongGraphType>>(
                nameof(Costume.RequiredBlockIndex),
                description: "Block index limit that locks the costume item. If the current block index is less than the RequiredBlockIndex, the item cannot be used."
            );
```

File: NineChronicles.Headless/GraphTypes/States/Models/Item/ShopItemType.cs (hunk 2 above)
Comment (area363) [inline NineChronicles.Headless/GraphTypes/States/Models/Item/ShopItemType.cs:42]:
```suggestion
            Field<NonNullGraphType<LongGraphType>>(
                nameof(ShopItem.ExpiredBlockIndex),
                description: "Block index limit that locks the shop item. If the current block index is greater than the ExpiredBlockIndex, the item cannot be traded.",
                resolve: context => context.Source.ExpiredBlockIndex
            );
```

File: schema.graphql [hunk 3]
```diff
@@ -306,6 +307,9 @@ type CostumeType {
 
   # Item category.
   itemType: ItemType!
+
+  # Block index at the costume can use.
```
Comment (area363) [inline schema.graphql:311]:
```suggestion
  # Block index limit that locks the costume item. If the current block index is less than the RequiredBlockIndex, the item cannot be used.
```

File: schema.graphql [hunk 4]
```diff
@@ -404,6 +408,9 @@ type ShopItemType {
   # Costume information.
   costume: CostumeType
 
+  # Block index at the shop item expiration.
```
Comment (area363) [inline schema.graphql:411]:
```suggestion
  # Block index limit that locks the shop item. If the current block index is greater than the ExpiredBlockIndex, the item cannot be traded.
```

File: schema.graphql (hunk 3 above)
Comment (moreal) [inline schema.graphql:311]: This file seems not updated after applying the review.

**Review by area363 — APPROVED** [review]
Please take care of [this part](https://github.com/planetarium/NineChronicles.Headless/pull/344#discussion_r590120096)  and you're good!

**Pull request merged**